### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/blog/timeline.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#613] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/blog/timeline.html

你可以通过 /timeline/ 访问时间页面，在页面中，你可以按照时间倒序查看所有的文章列表。 时间线列表 时间线列表 排除文章 默认情况下所有注明了写作日期的文章都会显示时间线中。如果你不希望某篇文章被包含，只需在文章的 frontmatter 中将 timeline 设置为 false。 配置时间线顶部文字 我们允许你定制时间线的顶部文字，你可以...

https://theme-hope.vuejs.press/zh/guide/blog/timeline.html

**Comment by Paper-Dragon** [comment]
您好 **作者**,

我遇到一个问题。

## 时间线只显示最新的一天
> https://paper-dragon.github.io/timeline


不清楚这个时间线的时间是怎么获取的，不是git commit time？
为啥我这个构建之后时间都变成了最新的一天？

### 猜测这里应该是一个bug。。

如果我在本地
```bash
pnpm run docs:dev
```
时间线显示为空

在本地  `pnpm run docs:build` 生成的时间线正常显示。

可能涉及到的其他信息
- 代码仓库： https://github.com/Paper-Dragon/paper-dragon.github.io
- 网页显示： https://paper-dragon.github.io/
- 浏览器Firefox，Chrome都会重现

## echarts的建议
希望能够出一个 echarts-wordcloud 的插件生成词云。我现在是使用在后端生成图片前端markdown获取。
希望能够直接以markdown 增强插件的形式直接前端动态生成词云。
现在在markdown增强里面只加载了echats.js的js文件，并没有扩展js，希望这个js能够扩展

可能用到的资料：
-  https://www.npmjs.com/package/echarts
- https://www.npmjs.com/package/echarts-wordcloud

## 希望增加的功能
- 类似于iframe ，在markdown里嵌入一个网页的markdown增强插件

如果有机会，请作者喝咖啡咖啡😚！！

期待能够得到作者回复。

**Reply by Paper-Dragon** [reply]
时间线问题已经解决，需要克隆整个仓库，克隆部分会检测不到。

**Comment by zxwin0125** [comment]
我这里时间线有个bug，某一个时间重复出现，但是这个时间点又没有文档记录
